qa-pipelines/config-capci.yml

```diff
@@ -0,0 +1,32 @@
+#Dev's upgrade CI: Test upgrades on nightly buids
+#Deploy: ./set-pipeline -t provo -p DEV-UPGRADES --pool=capci pipeline-presets/cap-qa-upgrades-lite.yml
+---
+src-branch: develop
+src-ci-repo: https://github.com/SUSE/cf-ci.git
+
+src-rails-example-repo: https://github.com/scf-samples/rails-example.git
+kube-pool-repo: git@github.com:SUSE/cf-ci-pools.git
+kube-pool-key: *github-private-key
+kube-pool-branch: cap-ci-kube-hosts
+kube-pool-pool: cap-ci-kube-hosts
+
+s3-config-endpoint: ~
+s3-config-access-key: *aws-access-key
+s3-config-secret-key: *aws-secret-key
+s3-config-bucket-opensuse: cap-release-archives
+s3-config-prefix-opensuse: nightly/
+
+s3-config-bucket-sles: cap-release-archives
+s3-config-prefix-sles: nightly/
```

Comment:
Currently I believe we don't produce SLES nightlies

Reply:
mm that's ok. we wont turn on that job for now. Ultimately, we need to test sles nightlies though :)

Reply:
Yes,  testing SLES nightlies is good, but we need to have them to test ;)